test/integration/integration_test.h

```diff
@@ -10,4 +10,17 @@ class IntegrationTest : public TestBaseWithParam<Network::Address::IpVersion>,
 public:
   IntegrationTest() : HttpIntegrationTest(Http::CodecClient::Type::HTTP1, GetParam(), realTime()) {}
 };
+
+class UpstreamEndpointIntegrationTest : public TestBaseWithParam<Network::Address::IpVersion>,
+                                        public HttpIntegrationTest {
+public:
+  UpstreamEndpointIntegrationTest()
+      : HttpIntegrationTest(Http::CodecClient::Type::HTTP1,
+                            /*upstream_address_fn=*/
```

Comment:
nit: I would remove this comment here and elsewhere. We don't typically do this.

Reply:
Done.